server/interpreter.js

```diff
@@ -4212,8 +4212,9 @@ Interpreter.prototype.installTypes = function() {
     // and we need to audit all use of String() throughout the
     // interpreter (including implicit use inside v8-native
     // functions).
-    return 'function ' + this.get('name', intrp.ANYBODY) +
-        '() { [native code] }';
+    var pd = this.getOwnPropertyDescriptor('name', intrp.ANYBODY);
+    var name = pd ? pd.value : '';
+    return 'function ' + name + '() { [native code] }';
```

Comment:
What happens if 'name' is an Object.create(null) or some other value that can't be stringified?  Is that possible?  If it is, Code City will crash here.